.azp/test.yml

```diff
@@ -1,8 +1,8 @@
 variables:
```

Comment:
Yikes, Azure seems to be running every single commit even though `batch` is supposed to be true by default. Try forcing the batch setting on to prevent the queue from overloading when there are many commits pushed in a short time.
```
trigger:
  batch: true
  branches:
    include:
    - '*'
```

Reply:
I was purposefully using the CI as a compiler ersatz, that's why it was busy